packages/cudos-utilities/nodeStatus.js

```diff
@@ -1,9 +1,9 @@
 const axios = require('axios').default
```

Comment:
Is it a good idea to move `nodeStatus.js` to `packages/cudos-cmd/node ` instead of `packages/cudos-utilities`? It is more relevant to general node commands than it is to all commands and also file/function naming is about the node.

Reply:
Probably we will use it as an utils at some point